hw/scripts/micro-trace-buffer.py

```diff
@@ -37,6 +37,12 @@ class MicroTraceBuffer(gdb.Command):
 
     def __init__(self):
         super(MicroTraceBuffer, self).__init__("mtb", gdb.COMMAND_STATUS, gdb.COMPLETE_NONE)
+        try:
+            mtb_state_at_crash = gdb.lookup_symbol("mtb_state_at_crash")[0]
```

Comment:
this can also detect mtb_state_at_crash variable on live system and it this case real addresses should be used

Reply:
fixed by checking content of mtb_state_at_crash not only variable existence